src/data-card-advanced.tsx

```diff
@@ -0,0 +1,498 @@
+    const marginTopTabletOrBigger = isTabletOrBigger ? '8px' : '';
+    const marginTopButton = isMobile ? '16px' : '8px';
+    const maxWidth = hasButtonLink && !hasAllItens ? '178px' : '';
+
+    return (
+        <div>
+            <div style={{marginLeft: -24, marginRight: -24}}>
+                <Divider />
+            </div>
+
+            <div
+                className={styles.actions}
+                style={{flexDirection, alignItems, marginTop: marginTopTabletOrBigger}}
+            >
+                {hasButton && (
+                    // eslint-disable-next-line jsx-a11y/no-static-element-interactions
+                    <div
+                        aria-label="button"
+                        tabIndex={-1}
+                        onClick={(event) => event.stopPropagation()}
+                        className={sprinkles({
+                            display: 'flex',
+                        })}
+                        style={{
+                            marginTop: marginTopButton,
+                            marginRight: '16px',
+                            position: 'relative',
+                            zIndex: '2',
+                        }}
+                    >
+                        {button}
+                    </div>
+                )}
+                <div
+                    style={{
+                        display: 'flex',
+                        flexDirection: 'row',
+                        marginTop,
+                        flexWrap: 'wrap',
+                        marginRight,
+                    }}
+                >
+                    {hasFooterImage && (
+                        <div
+                            style={{marginRight: '16px', zIndex: '0'}}
+                            className={sprinkles({alignItems: 'center', display: 'flex'})}
+                        >
+                            {footerImage}
+                        </div>
+                    )}
+
+                    {hasFooterText && (
+                        <div style={{maxWidth}} className={styles.footerText}>
+                            <Text2 truncate={footerTextLinesMax} regular>
+                                {footerText}
+                            </Text2>
+                        </div>
+                    )}
+                </div>
+                {hasButtonLink && (
+                    // eslint-disable-next-line jsx-a11y/no-static-element-interactions
+                    <div
+                        aria-label="link"
+                        tabIndex={-1}
+                        className={sprinkles({
+                            display: 'flex',
+                        })}
+                        onClick={(event) => event.stopPropagation()}
+                        style={{
+                            marginTop: isMobile ? marginTop : marginTopButton,
+                            position: 'relative',
+                            zIndex: '2',
```

Comment:
you have too many zindex. can you explain why?

Reply:
Before the zIndex was used to solve problems with clickable areas in the data-card, but as it is no longer a problem, I removed them except one, which is for the top actions